Source/relay/TourGuide/Paths/Avatar of Shadows over Loathing.ash

```diff
@@ -0,0 +1,25 @@
+RegisterTaskGenerationFunction("PathShadowsOverLoathingGenerateTasks");
+void PathShadowsOverLoathingGenerateTasks(ChecklistEntry [int] task_entries, ChecklistEntry [int] optional_task_entries, ChecklistEntry [int] future_task_entries)
+{
+    if (lookupSkill("Free-For-All").have_skill()) {
+        if ($effect[Everything Looks Red].have_effect() > 0) return;
+        string [int] description;
+        string main_title = HTMLGenerateSpanFont("Free-For-All castable", "red");
+        description.listAppend("Free instakill (25 adv cooldown)");
+        task_entries.listAppend(ChecklistEntryMake("__item orange boxing gloves", "", ChecklistSubentryMake(main_title, "", description), -11));
+    }
+    else if (lookupSkill("Fondeluge").have_skill()) {
```

Comment:
we should also add a path toggle on the parka YR tile to make sure this tile has precedence if you have fondeluge; this is a much better YR than anything else they'd use. maybe we should take all the ELY tiles and turn them into one big ELY tile that shows the "best" available YR source if ELY is not active and the user has the ability to do a YR?

Reply:
@docrostov Hmm, I thought there was some parka tile for spitting acid too but I don't think there actually is one. Removed the fireworks and I think that's all that is needed here, but let me know if you notice something I'm missing.